pkgs/development/python-modules/btchip/default.nix

```diff
@@ -0,0 +1,20 @@
+{ stdenv, buildPythonPackage, fetchPypi, hidapi, pyscard, ecdsa }:
+
+buildPythonPackage rec {
+  pname = "btchip-python";
+  version = "0.1.21";
+  name = "${pname}-${version}";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "16g9l3rpxpvvkdx08mgy0ligvsfcpzdrh4hplj104cpprrbsqd6v";
+  };
+
+  buildInputs = [ hidapi pyscard ecdsa ];
+
+  meta = with stdenv.lib; {
+    description = "Python communication library for Ledger Hardware Wallet products";
+    homepage = "https://github.com/LedgerHQ/btchip-python";
+    license = licenses.apache2;
```

Comment:
Use `licenses.asl20`. You can test meta eval by `nix-instantiate --eval --strict -A foo.meta`.